### PR TITLE
feature: completion

### PR DESCRIPTION
**Pull request by snowhork**
This MR is for completion feature.

### CODE REVIEW

**Comment by snowhork** [comment]
Please review.

**Comment by snowhork** [comment]
@hirosassa please review.

File: README.md [hunk 1]
```diff
@@ -70,3 +70,27 @@ Permit user2@email.com to bq-project-id access as READER
...
+
+### Updating data
+```
+bqiam completion
+```
+
+
+### Limitaion
+Completion data file is placed on `~/.bqiam-completion-file.toml`.
+This path is unchangeable.
```
Comment (hirosassa) [inline README.md]: nice doc!

File: bqrole/dataset.go [hunk 2]
```diff
@@ -84,7 +84,8 @@ func PermitDataset(role bq.AccessRole, project string, users, datasets []string)
 
 // grantBQRole grants user roles/bigquery permission
 func grantBQRole(project, user string, role string) error {
-	policy, err := fetchCurrentPolicy(project)
+	policy, err := FetchCurrentPolicy(project)
+
```
Comment (hirosassa) [inline bqrole/dataset.go]: [nits] IMO, this blank line is unnecessary (not consistent with [here](https://github.com/hirosassa/bqiam/pull/32/files#diff-d6cbf47806cb27b6da400d2a751f05527f57d2c49ddc7f1727ccbab554191a63R62))

**Review by hirosassa — COMMENTED** [review]
Thanks for your contribution! I added a nits comment.

**Comment by snowhork** [comment]
@hirosassa sorry to be late.

I have fixed the completion file path limitation. Now, we can specify the path by `.bqiam.toml`

please review.

**Comment by hirosassa** [comment]
LGTM! Thanks for your GREAT work!

File: README.md [hunk 3]
```diff
@@ -70,3 +70,22 @@ Permit user2@email.com to bq-project-id access as READER
...
+Download projects, datasets, users list data via GCP API.
+
+### bash
+```
+bqiam completion bash > /path/to/bash-completion/completions/bqiam
+```
+
+### zsh
+```
+bqiam completion bash > /path/to/zsh-completions/_bqiam
```
Comment (kitagry) [inline README.md:85]: `bqiam completion zsh` ?

Comment (hirosassa) [reply README.md:85]: @kitagry I fixed it on the PR below! Please check
https://github.com/hirosassa/bqiam/pull/39

Comment (kitagry) [reply README.md:85]: :+1:

**Pull request merged**